Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -210,81 +211,105 @@ public Guid Guid
             }
         }
 
-        public void SaveBackup()
+        public async Task SaveBackupAsync()
+        {
+            await Task.Run(() => 
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)